internal/data/opentelemetry-proto-gen/collector/logs/v1/logs_service.pb.go

```diff
@@ -6,14 +6,16 @@ package v1
 import (
 	context "context"
 	fmt "fmt"
+	io "io"
```

Comment:
Why is this file changed? Did you regenerate the protobufs?

Reply:
`goimports` changed it since standard libraries should go in the first group of imports.

Reply:
I think we need to prohibit goimport from touching generated files. I haven't seen happening the past, not sure why goimport is doing it now.